docs/en/overview.md

```diff
@@ -1,3 +1,49 @@
 # Overview
 
 - [Overview](#overview)
+  - [Introduction of Self-supervised Learning](#introduction-of-self-supervised-learning)
+  - [Design of MMSelfSup](#design-of-mmselfsup)
+  - [Hands-on Roadmap of MMselfSup](#hands-on-roadmap-of-mmselfsup)
+
+In this section, We would like to give a quick review of the open-source library [MMSelfSup](https://github.com/open-mmlab/mmselfsup).
+
+We will first illustrate the basic idea of the self-supervised learning, then we will briefly describe the design of MMSelfSup. After that, we will provide a hands-on roadmap to help the users to play with MMSelfSup
+
+## Introduction of Self-supervised Learning
+
+Self-supervised learning(SSL) is a promising learning paradigm, which aims to leverage the potential of the huge amount of unlabeled data. In SSL, we typically uses the label generated automatically without human labor, to learn a model to extract the discriminative representation of the data. Equipped with the powerful pre-trained model by SSL, we are able to improve various downstream vision tasks currently.
+
+The community has witnessed rapid development of SSL in the past few years. Our codebase aims to become an easy-to-use and user-friendly library, to help the research and engineering. We will elaborate the properties and design of MMSelfSup in the following sections.
+
+## Design of MMSelfSup
+
+MMSelfSup follows the modular designed architecture as other OpenMMLab projects. We illustrate the overall framework is illustrated as below:
+
+<div align="center">
+  <img src="../../resources/mmselfsup_arch_uml.jpg" width="500"/>
```

Comment:
Better use a url, instead of a local link, for keeping the image locally will increase the size of this repo.